app/uk/gov/hmrc/helptosavefrontend/views/register/bank_account_details.scala.html

```diff
@@ -0,0 +1,65 @@
+@*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.helptosavefrontend.forms.BankDetails
+@import uk.gov.hmrc.helptosavefrontend.views.html.helpers
+@import uk.gov.hmrc.helptosavefrontend.models.HtsContext
+@import uk.gov.hmrc.helptosavefrontend.config.FrontendAppConfig
+@import uk.gov.hmrc.helptosavefrontend.views.helpers.errorPrefix
+@import uk.gov.hmrc.helptosavefrontend.controllers.routes
+@import uk.gov.hmrc.helptosavefrontend.config.FrontendAppConfig
+
+@(form: Form[BankDetails], referrer: String)(implicit htsContext: HtsContext, request: Request[_], messages: Messages, appConfig: FrontendAppConfig)
+
+@uk.gov.hmrc.helptosavefrontend.views.html.main_template(title = "Enter Bank Details", bodyClasses = None) {
+@helpers.form(routes.BankAccountController.submitBankDetails(), 'id -> "bankDetailsForm") {
+
+<a href="@referrer">Back</a>
```

Comment:
not sure we'll need the referrer here - should be able to get the previous page via the browser history, e.g. https://www.w3schools.com/jsref/met_his_back.asp